dotcom-rendering/src/lib/ArticleRenderer.tsx

```diff
@@ -66,6 +66,7 @@ export const ArticleRenderer = ({
 	abTests,
 	editionId,
 }: Props) => {
+	const totalElements = elements.length;
 	const renderedElements = elements.map((element, index) => {
```

Comment:
You could do this inline with the [third parameter of the `map` callback](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#array):

```ts
const renderedElements = elements.map((element, index, { length } ) => {
  // do stuff
}
```